### DISCUSSION TITLE
Connecting 2n with Axis Companion

### DISCUSSION THREAD

**Discussion by TvOESB** [AxisCommunications/discussions#12] [General] [answered]
Hi together,

our customer have a few cameras from Axis. He uses the Axis Companion App etc. 
Now he wants a Intercomsystem in the building.
I know Axis also offers Intercom Systems, but only with one push button.
We need 3 push buttons for the building.
Is ist possible to install a intercom system of 2n and use the cameras in the Companion App?
Maybe i can mount the camera of 2n directly to the Companion App or can i implement a RTSP Stream from a different camera to the Companion App?

**Comment by mattias-kindborg-at-work** [comment]
Hi @TvOESB! I'll see if I can find the answer.

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
Sorry no, AXIS Companion only supports Axis branded intercoms.

Depending on your environment, migrating to [AXIS Camera Station](https://www.axis.com/products/axis-camera-station) would enable you to [support 2B](https://www.axis.com/files/tech_notes/How_to_2N_Intercom_AXIS_Camera_Station_en.pdf).

**Reply by TvOESB** [reply]
Thanks for the answer!
